horreum-api/src/main/java/io/hyperfoil/tools/horreum/api/services/TestService.java

```diff
@@ -228,18 +230,20 @@ List<ExportedLabelValues> listLabelValues(@PathParam("id") int testId,
 
    @GET
    @Path("{id}/export")
-   @APIResponseSchema(value = String.class,
-           responseDescription = "A Test defintion formatted as json",
+   @Produces(MediaType.APPLICATION_JSON)
+   @APIResponseSchema(value = TestExport.class,
+           responseDescription = "A Test definition formatted as json",
            responseCode = "200")
-   String export(@PathParam("id") int testId);
+   TestExport export(@PathParam("id") int testId);
 
    @POST
    @Path("import")
-   @APIResponse(responseCode = "204", description = "Import a new test")
-   @RequestBody(content = @Content( mediaType = MediaType.APPLICATION_JSON,
-           schema = @Schema( type = SchemaType.STRING, implementation = String.class)) )
-   @Operation(description="Import a previously exported Test")
-   void importTest( String testConfig);
+   @Consumes(MediaType.APPLICATION_JSON)
+   @APIResponse(responseCode = "204", description = "Import a new Test or update an existing Test")
```

Comment:
This response code should be "201": https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/201